lib/ansible/utils/display.py

```diff
@@ -128,12 +129,15 @@ def display(self, msg, color=None, stderr=False, screen_only=False, log_only=Fal
                 # characters that are invalid in the user's locale
                 msg2 = to_unicode(msg2, self._output_encoding(stderr=stderr))
 
-            if not stderr:
-                sys.stdout.write(msg2)
-                sys.stdout.flush()
-            else:
-                sys.stderr.write(msg2)
-                sys.stderr.flush()
+            fileobj = sys.stderr if stderr else sys.stdout
```

Comment:
no trailing conditionals, people tend to miss these (i'm guilty of having written too many of em)